src/Illuminate/Queue/QueueManager.php

```diff
@@ -233,6 +234,16 @@ public function setDefaultDriver($name)
     }
 
     /**
+     * Get the name of the queue prefix.
+     *
+     * @return string
+     */
+    public function getQueuePrefix()
+    {
+        return isset($this->app['config']['queue.prefix']) ? $this->app['config']['queue.prefix'] : null;
```

Comment:
`$this->app['config']` must return object. Object of class `Illuminate\Config\Repository`